docs/LoaderDriverInterface.md

```diff
@@ -842,7 +843,9 @@ function.
 
 If a driver does implement this support, it must export the function from the
 driver library using the name `vk_icdGetPhysicalDeviceProcAddr` so that the
-symbol can be located through the platform's dynamic linking utilities.
+symbol can be located through the platform's dynamic linking utilities, or if
+the driver supports Loader Version 7, simply be exposed through
```

Comment:
I think it would be a little clearer if "simply be exposed" was modified with "the function may be exposed".

Reply:
This has been fixed, though with slightly different language.
Please re-review the newest version.